README.md

```diff
@@ -622,6 +623,30 @@ Floating-point numbers should always include a leading and/or trailing zero:
 3.f0
 ```
 
+### Ternary Operator
+
+Ternary operators should generally only consume a single line.
```

Comment:
```suggestion
Ternary operators should generally only consume a single line.
Do not chain multiple ternary operators.
```

Reply:
I am dubious sometimes it is the best way.
Maybe: Think twice before chaining ternery operators,
and consider if using dispatch, or a dictionary might be better,

Reply:
well, those seem like performance questions.

style-wise (i.e. when what you want is 1 `if ... elseif ... elseif ...`) we do in fact always say 
```
# Yes:
foobar = if foo == 2
    bar
elseif foo == 3
    qux
else
    baz
end

# No:
foobar = foo == 2 ? bar : foo == 3 ? qux : baz
```

Reply:
Is there a performance difference between the two examples I give?

Reply:
Or is your point about clarity when there are many conditions?

We could add: "If chaining many conditions, consider if using dispatch or a dictionary would be clearer."

Reply:
Happy to discuss wording/examples here, but if we don't think this is already a settled matter then let's leave it to another PR, and merge this one as-is. :)

Reply:
You are correct.

Reply:
How's this?

```suggestion
Ternary operators should generally only consume a single line.
Do not chain multiple ternary operators.
If chaining many conditions, consider using dispatch or a dictionary.
```

Reply:
fine

Reply:
Or even leave it with your original suggestion

Reply:
👍

Reply:
Keeps this inline with how the official docs talk about this:  https://docs.julialang.org/en/v1/manual/control-flow/#Control-Flow-1
```suggestion
Ternary operators (`?:`) should generally only consume a single line.
```